new-site/src/components/Playground.scss

```diff
@@ -0,0 +1,106 @@
+@charset "utf-8";
+
+@import '~hds-design-tokens/lib/all.scss';
+
+.playground-block {
+  --focus-border-width: 2px;
+  margin: var(--spacing-l) 0;
+}
+
+.playground-block-tabs {
+  --tablist-border-color: transparent;
```

Comment:
I guess this overrides the `--tablist-border-color` variable defined in Tabs.module.scss? Does it depend on the css order for which one of them takes precedence over another?

Reply:
I believe it overrides the root value when the property is given in the element scope 🤔